### ISSUE TITLE
Quake = Free on Epic = Missing files

### ISSUE THREAD

**Issue by chaosophy** [sultim-t/vkquake-rt#26]
As a heads up Quake is free on the Epic store this week!

I downloaded the quake-rt-1_0_2 AMD release, unzipped and followed the manual install path.

Copied...
D:\Program Files (x86)\Epic Games\Quake\id1\pak0.pak
D:\Program Files (x86)\Epic Games\Quake\id1\music
to an id1 folder next to vkQuake,exe
and also copied...
D:\Program Files (x86)\Epic Games\Quake\QuakeEX.kpf
next to the vkQuake.exe

but when I run vkQuake I get the following pop up...

---------------------------
Missing files
---------------------------
Some files in a local "id1" folder are missing.
Copy them from a Steam folder?
---------------------------
Yes   No   
---------------------------

game still seems to load fine after clicking No.

Wondering if anyone might know what it thinks is missing?  
And if there are any more files I need to copy over?
Not sure if I've done something wrong, the install notes are wrong, or if the app has something odd hardcoded?

Thanks.

**Comment by HarbingerDawn** [comment]
It always seems to say that when using the files from the re-release rather than classic Quake. It doesn't seem to affect anything, the pop-up just needs to be reprogrammed to only occur if classic files are used instead of files from the re-release.

**Comment by chaosophy** [comment]
Hi, thanks for confirming, was worried I'd messed it up. 
I guess it's looking for the pak1.pak file or something?

**Comment by Andrew-Cottrell** [comment]
This message also appears when using classic Quake. I get the same '**Some files in a local "id1" folder are missing**' message even though I have both the `ID1\PAK0.PAK` and `ID1\PAK1.PAK` files — each with creation date Friday, ‎12 ‎July ‎1996, ‏‎18:31:52 — copied from the original CD-ROM disc. I just click "No" every time and the game works fine (I have completed the game on this engine). Perhaps the engine's detection routine is looking for a Steam-version specific file that isn't actually needed.

**Comment by chaosophy** [comment]
Ah yes, my bad, it was from the Prime offer.  I must of got confused as it redeems to Epic store and I had been on Epic yesterday to claim the weekly free games on there.

Also, thanks for confirming you see the same issue as my op.

**Comment by HarbingerDawn** [comment]
> I get the same '**Some files in a local "id1" folder are missing**' message even though I have both the `ID1\PAK0.PAK` and `ID1\PAK1.PAK` files

Did you also copy the music to /id1/music/?

**Comment by Andrew-Cottrell** [comment]
> Did you also copy the music to /id1/music/?

Yes, I also have `ID1\music` containing 10 files named `Track02.ogg` through to `Track011.ogg`. These contain Vorbis 44100Hz stereo 256 kbit/sec streams. The music plays in game.

**Comment by Berzerk2k2** [comment]
The "Some files in a local "id1" folder are missing" message stopped appearing after I once hit the YES button.

Quake RT then copied the pak files (0 and 1) from the classic id1 folder (not the one in "rerelease") into the id1 folder that's located in my Quake RT installation. Now when I start the game the message doesn't show up anymore.

Looks like the rerelease files (pak0.pak, QuakeEX.kpf) are not recognised/supported by QRT at the moment so when you use any of those the message will appear.

**Comment by Andrew-Cottrell** [comment]
I have fixed my install to avoid the message. The solution is to ensure your folders & files match the lists specified in

https://github.com/sultim-t/vkquake-rt/blob/1.20.2-rt1.0.2/Quake/common.c#L2230

```c
static const wchar_t *rt_folderstocreate[] = {
    L"\\id1",
    L"\\id1\\music",
};
static const wchar_t *rt_originalfiles[] = {
	L"\\id1\\PAK0.PAK",
	L"\\id1\\PAK1.PAK",
	L"\\rerelease\\id1\\music\\track02.ogg",
	L"\\rerelease\\id1\\music\\track03.ogg",
	L"\\rerelease\\id1\\music\\track04.ogg",
	L"\\rerelease\\id1\\music\\track05.ogg",
	L"\\rerelease\\id1\\music\\track06.ogg",
	L"\\rerelease\\id1\\music\\track07.ogg",
	L"\\rerelease\\id1\\music\\track08.ogg",
	L"\\rerelease\\id1\\music\\track09.ogg",
	L"\\rerelease\\id1\\music\\track10.ogg",
	L"\\rerelease\\id1\\music\\track11.ogg",
};
static const wchar_t *rt_dstfiles[] = {
	L"\\id1\\PAK0.PAK",
	L"\\id1\\PAK1.PAK",
	L"\\id1\\music\\track02.ogg",
	L"\\id1\\music\\track03.ogg",
	L"\\id1\\music\\track04.ogg",
	L"\\id1\\music\\track05.ogg",
	L"\\id1\\music\\track06.ogg",
	L"\\id1\\music\\track07.ogg",
	L"\\id1\\music\\track08.ogg",
	L"\\id1\\music\\track09.ogg",
	L"\\id1\\music\\track10.ogg",
	L"\\id1\\music\\track11.ogg",
};
```

A helpful enhancement would be to include this information in either [readme.md](https://github.com/sultim-t/vkquake-rt/blob/1.20.2-rt1.0.2/readme.md) or the [release notes](https://github.com/sultim-t/vkquake-rt/releases/tag/1.20.2-rt1.0.2).

**Comment by TR2NLG** [comment]
Is this the same problem / folder circumstance that causes the Rerelease assests don´t appear ingame?

**Comment by Laithan401** [comment]
Same issue here.. cannot get RT to work with the rerelease no matter what I do. It seems to be hardcoded to use files from the base steam folder and not using the rerelease folder. I even put the RT file directly into the rerelease folder and it still loads the original version.
